codalab_service.py

```diff
@@ -321,9 +342,20 @@ def has_callable_default(self):
 for worker_manager_type in ['cpu', 'gpu']:
     CODALAB_ARGUMENTS += [
         CodalabArg(
-            name='worker_manager_{}_queue'.format(worker_manager_type),
-            help='Name of queue to submit {} jobs'.format(worker_manager_type),
-            default='codalab-{}'.format(worker_manager_type),
+            name='worker_manager_{}_default_cpus'.format(worker_manager_type),
+            type=int,
+            default=1,
+            help='Default number of CPUs for each worker started by the {} worker manager'.format(
+                worker_manager_type
+            ),
+        ),
+        CodalabArg(
+            name='worker_manager_{}_default_memory_mb'.format(worker_manager_type),
+            type=int,
+            default=1024,
```

Comment:
this default doesn't match the 2048 above---is that intentional?

Reply:
I arbitrarily set the value. I will update to be consistent with the worker managers.